File: WebApp/Controllers/CustomControllers/MBS/Models/AchievementSummary.cs
using System;

namespace WebApi.Models.MBS
{
    public class AchievementSummaryDTO
    {
        public Guid ProjectId { set; get; }
        public Guid AchievementId { set; get; }
        public int NumberOfTargets { set; get; }
        public int NumberOfEvidenceItems { set; get; }
        public bool IsAchieved { set; get; }
        public string BackgroundColor { set; get; }
    }
}

File: WebApp/wwwroot/scripts/domain/custom/MBS/achievementSummary.ts

module App.Models.MBS {
    "use strict";

    export interface IAchievementSummary extends ng.resource.IResource<IAchievementSummary> {
        projectId: string;
        achievementId: string;
        numberOfTargets: number;
        numberOfEvidenceItems: number;
    }
}